Examples/BinaryBHCubicHorndeski/params.txt

```diff
@@ -149,24 +146,23 @@ lapse_advec_coeff = 1.0
 lapse_coeff = 2.0
 lapse_power = 1.0
 
-# Shift evolutions
-shift_advec_coeff = 0 # Usually no advection for beta
+# Shift evolution
+shift_advec_coeff = 0.0 # Usually no advection for beta
 shift_Gamma_coeff = 0.75
 eta = 1.0 # eta of gamma driver, should be of order ~1/M_ADM of spacetime
 
 # CCZ4 parameters
-formulation = 0  # 1 for BSSN, 0 for CCZ4
-kappa1 = 0.1
-kappa2 = 0.
+kappa1 = 1.0
+kappa2 = -0.1
 kappa3 = 1.
 covariantZ4 = 1 # 0: keep kappa1; 1 [default]: replace kappa1 -> kappa1/lapse
 
 # Modified CCZ4 parameters
 a0 = 0.2
 b0 = 0.4
 
-# Cubic Horndeski parameters
-g2 = 0.
+# 4dST parameters
```

Comment:
Think this should be cubic horndeski :-)

Reply:
Oh yes I'll correct it